bundle/edu.gemini.itc.shared/src/main/java/edu/gemini/itc/shared/SourceDefinition.java

```diff
@@ -118,7 +118,9 @@ public double getBBTemp() {
     }
 
     public Wavelength getELineWavelength() {
-        return ((EmissionLine) distribution).wavelength();
+        // Wavelength is a value class, on the Java side wavelength() therefore returns just a Length
+        // and not a Wavelength object, so we need to repackage it to make it a wavelength. Blech.
+        return new Wavelength(((EmissionLine) distribution).wavelength());
```

Comment:
Yo! This is a funny one where Java doesn't know what Scala is doing. The remedy will be to turn `SourceDefinition` into a Scala case class sometime in the future. Baby steps, baby steps...